--- origsrc/libsndfile-1.0.17/configure.ac	2008-12-23 01:07:51.521750000 -0600
+++ src/libsndfile-1.0.17/configure.ac	2008-12-23 01:38:46.756125000 -0600
@@ -308,10 +308,6 @@
 		use_windows_api=1
 		OS_SPECIFIC_LINKS="-lwinmm"
 		;;
-	cygwin*)
-		os_is_win32=1
-		OS_SPECIFIC_LINKS="-lwinmm"
-		;;
 	esac
 
 AC_DEFINE_UNQUOTED(OS_IS_WIN32, ${os_is_win32}, [Set to 1 if compiling for Win32])
--- origsrc/libsndfile-1.0.17/examples/sndfile-play.c	2006-08-31 04:22:07.000000000 -0500
+++ src/libsndfile-1.0.17/examples/sndfile-play.c	2008-12-23 01:26:11.959250000 -0600
@@ -34,7 +34,7 @@
 	#include <sys/time.h>
 #endif
 
-#if defined (__linux__)
+#if defined (__linux__) || defined(__CYGWIN__)
 	#include 	<fcntl.h>
 	#include 	<sys/ioctl.h>
 	#include 	<sys/soundcard.h>
@@ -339,7 +339,7 @@
 **	Linux/OSS functions for playing a sound.
 */
 
-#if defined (__linux__)
+#if defined (__linux__) || defined(__CYGWIN__)
 
 static	int	linux_open_dsp_device (int channels, int srate) ;
 
@@ -926,7 +926,7 @@
 		return 1 ;
 		} ;
 
-#if defined (__linux__)
+#if defined (__linux__) || defined(__CYGWIN__)
 	#if HAVE_ALSA_ASOUNDLIB_H
 		if (access ("/proc/asound/cards", R_OK) == 0)
 			alsa_play (argc, argv) ;
--- origsrc/libsndfile-1.0.17/src/file_io.c	2006-08-31 04:22:07.000000000 -0500
+++ src/libsndfile-1.0.17/src/file_io.c	2008-12-23 01:24:58.443625000 -0600
@@ -545,7 +545,7 @@
 				break ;
 		} ;
 
-#if OS_IS_WIN32
+#if OS_IS_WIN32 || defined(__CYGWIN__)
 	/* For Cygwin. */
 	oflag |= O_BINARY ;
 #endif
