--- origsrc/libsndfile-1.0.25/configure.ac	2015-11-09 10:25:02.495815900 -0600
+++ src/libsndfile-1.0.25/configure.ac	2015-11-09 10:34:29.723844700 -0600
@@ -335,7 +335,7 @@ if test -n "$PKG_CONFIG" ; then
 		enable_external_libs=yes
 
 		EXTERNAL_CFLAGS="$FLAC_CFLAGS $OGG_CFLAGS $VORBISENC_CFLAGS $SPEEX_CFLAGS"
-		EXTERNAL_LIBS="$FLAC_LIBS $VORBISENC_LIBS $SPEEX_LIBS"
+		EXTERNAL_LIBS="$FLAC_LIBS $VORBISENC_LIBS $VORBIS_LIBS $OGG_LIBS $SPEEX_LIBS"
 	else
 		echo
 		AC_MSG_WARN([[*** One or more of the external libraries (ie libflac, libogg and]])
