--- origsrc/libsndfile-1.0.19/programs/sndfile-play.c
+++ src/libsndfile-1.0.19/programs/sndfile-play.c
@@ -34,7 +34,7 @@
 	#include <sys/time.h>
 #endif
 
-#if defined (__linux__)
+#if defined (__linux__) || defined(__CYGWIN__)
 	#include 	<fcntl.h>
 	#include 	<sys/ioctl.h>
 	#include 	<sys/soundcard.h>
@@ -339,7 +339,7 @@
 **	Linux/OSS functions for playing a sound.
 */
 
-#if defined (__linux__)
+#if defined (__linux__) || defined(__CYGWIN__)
 
 static	int	linux_open_dsp_device (int channels, int srate) ;
 
@@ -926,7 +926,7 @@
 		return 1 ;
 		} ;
 
-#if defined (__linux__)
+#if defined (__linux__) || defined(__CYGWIN__)
 	#if HAVE_ALSA_ASOUNDLIB_H
 		if (access ("/proc/asound/cards", R_OK) == 0)
 			alsa_play (argc, argv) ;
