--- origsrc/libsndfile-1.0.17/configure.ac	2007-10-08 11:49:36.593750000 -0500
+++ src/libsndfile-1.0.17/configure.ac	2007-10-08 11:49:39.718750000 -0500
@@ -305,10 +305,6 @@
 		use_windows_api=1
 		OS_SPECIFIC_LINKS="-lwinmm"
 		;;
-	cygwin*)
-		os_is_win32=1
-		OS_SPECIFIC_LINKS="-lwinmm"
-		;;
 	esac
 
 AC_DEFINE_UNQUOTED(OS_IS_WIN32, ${os_is_win32}, [Set to 1 if compiling for Win32])
@@ -392,7 +388,7 @@
 if test $prefix = "NONE" ; then
 	htmldocdir=/usr/local/share/doc/libsndfile1-dev/html
 else
-	htmldocdir=$prefix/share/doc/libsndfile1-dev/html
+	htmldocdir=$prefix/share/doc/libsndfile-${VERSION}/html
 	fi
 
 if test x$enable_bow_docs = "xyes" ; then
--- origsrc/libsndfile-1.0.17/src/file_io.c	2006-08-31 04:22:07.000000000 -0500
+++ src/libsndfile-1.0.17/src/file_io.c	2007-10-08 11:49:39.718750000 -0500
@@ -545,7 +545,7 @@
 				break ;
 		} ;
 
-#if OS_IS_WIN32
+#ifdef __CYGWIN__
 	/* For Cygwin. */
 	oflag |= O_BINARY ;
 #endif
